cmake/templates/python_distutils_install.bat.in

```diff
@@ -16,4 +16,5 @@ set "CATKIN_BINARY_DIR=@CMAKE_BINARY_DIR@"
     build --build-base "@CMAKE_CURRENT_BINARY_DIR@" ^
     install %DESTDIR_ARG% @SETUPTOOLS_ARG_EXTRA@ ^
     --prefix="@SETUPTOOLS_INSTALL_PREFIX@" ^
-    --install-scripts="@SETUPTOOLS_INSTALL_PREFIX@\@CATKIN_GLOBAL_BIN_DESTINATION@"
+    --install-scripts="@SETUPTOOLS_INSTALL_PREFIX@\@CATKIN_GLOBAL_BIN_DESTINATION@" ^
+    --root="/"
```

Comment:
I am surprised that it should be `/` on Windows?

Reply:
@dirk-thomas Both (`/` and `\`) work here but for the consistency I changed it to backslash.

Reply:
Interesting - I wasn't even thinking slash vs. backslash. I was wondering what either does on Windows without a drive letter...